.hlint.yaml

```diff
@@ -66,6 +103,13 @@
         , note: "'sortWith' will be faster here because it doesn't do caching"
         }
 
+- hint: {lhs: "foldl' (flip f)", rhs: "flipfoldl' f"}
+
+- warn: {lhs: f >>= guard, rhs: guardM}
+- warn: {lhs: guard =<< f, rhs: guardM}
+
+- hint: {lhs: fmap (fmap f x), rhs: f <<$>> x}
```

Comment:
This is actually `fmap (fmap f) x`, not `fmap (fmap f x)`. Here is the step-by-step transformation:

1. `fmap . fmap`
2. `\a -> fmap (fmap a)`
3. `\a b -> fmap (fmap a) b`